client/src/js/modules/shipment/views/containers.js

```diff
@@ -141,12 +159,16 @@ define(['marionette',
         },
 
         updateCols: function(selected) {
-            var isPuck = (selected == null || selected == 'puck')
+            var isPuck = !this.imager && (selected == null || selected == 'puck')
```

Comment:
```suggestion
            var isPuck = !this.imager && (selected === null || selected === 'puck')
```

Might be useful to use strict equality here - since `null` == `undefined` in non-strict mode